src/unk_0203E880.c

```diff
@@ -593,19 +593,19 @@ static u32 sub_0203F110 (int param0)
     return MapHeader_GetMsgArchiveID(param0);
 }
 
-u16 * sub_0203F118 (FieldSystem * fieldSystem, u16 param1)
+u16 * sub_0203F118 (FieldSystem * fieldSystem, u16 varID)
```

Comment:
[question] Can this function be documented? It looks like a basic Variable accessor (that `0x8000` at the bottom leads me to think as much, anyways).